js/site.js

```diff
@@ -151,23 +154,23 @@ $("#find").submit(function (e) {
 
         if (!circleBoundsVisible) {
           // check if marker is inside the bounding box
-          isInsideRegion = findme_boundingBox._bounds.contains(dragMarkerLocation);
+          isInsideRegion = findMe_boundingBox._bounds.contains(dragMarkerLocation);
         } else {
           // check if marker is inside the circle
           isInsideRegion = isInsideCircle(dragMarkerLocation);
         }
 
         // reset marker to previous position when dragged outside the active bounding box
         if (!isInsideRegion){
-          findme_marker.setLatLng(activeMarkerLatLng);
+          findMe_marker.setLatLng(activeMarkerLatLng);
         }
       });
 
       /**
        * Validate new marker location (Marker "drag ended" event)
        * @param {Object} dragged_event
        */
-      findme_marker.on('dragend', function (dragged_event) {
+      findMe_marker.on('dragend', function (dragged_event) {
```

Comment:
Can you please undo the change to capitalization here? It's not necessary and is distracting from the actual logic changes below.